Sources/ProjectDescription/Config.swift

```diff
@@ -24,8 +34,11 @@ public struct Config: Codable, Equatable {
         /// Tuist disables echoing the ENV in shell script build phases
         case disableShowEnvironmentVarsInScriptPhases
 
+        /// When passed, Tuist will enable code coverage for autogenerated default schemes according to given mode
+        case enableCodeCoverage(CodeCoverageMode)
+
         /// When passed, Tuist will enable code coverage for autogenerated default schemes
-        case enableCodeCoverage
+        static let enableCodeCoverage = GenerationOptions.enableCodeCoverage(.all)
```

Comment:
I believe you'll have to make this one public in order for users to be able to use it from their manifests.